products/keycloak.md

```diff
@@ -154,6 +154,11 @@ Only the latest major version gets active development and security fixes.
 Depending on the severity of a vulnerability, the issue may be fixed in the current release or, for
 lower severity vulnerabilities or hardening, in the following release.
 
+The Keycloak JS adapter
+[split from Keycloak with the 26.2.0 release](https://www.keycloak.org/2025/02/keycloak-js-2620-released.html)
+and now follows its own release policy, and will continue to be backwards compatible with all actively
+supported releases of the Keycloak server, and deviation from this will be considered a breaking change.
```

Comment:
```suggestion
The Keycloak JS adapter
[split from Keycloak with the 26.2.0 release](https://www.keycloak.org/2025/02/keycloak-js-2620-released.html)
and now follows its own release policy. It will continue to be backwards compatible with all actively
supported releases of the Keycloak server, and deviation from this will be considered a breaking change.
```